apps/src/lib/ui/accounts/ManageLinkedAccounts.jsx

```diff
@@ -161,21 +171,18 @@ class ManageLinkedAccounts extends React.Component {
         id={authOption.id}
         email={this.formatEmail(authOption)}
         credentialType={authOption.credentialType}
-        disabledStatus={
-          authOption.id
-            ? this.disconnectDisabledStatus(authOption)
-            : this.connectDisabledStatus(authOption)
-        }
+        disabledStatus={this.disabledStatus(authOption)}
         error={authOption.error}
         personalAccountLinkingEnabled={this.props.personalAccountLinkingEnabled}
       />
     );
   };
 
   render() {
-    const lockedOptions = this.formatAuthOptions().filter(option => {
-      return this.shouldDisableConnectButton(option);
-    });
+    const [lockedOptions, unlockedOptions] = _.partition(
```

Comment:
Thanks for indulging my nitpick, and nice use of underscore! I always forget that we use it as our utility library of choice.

Reply:
Yep nice to learn about the partition method! I was about to write it myself before coming to my senses and checking.